include/smack/SmackRep.h

```diff
@@ -83,7 +89,16 @@ class SmackRep {
   Program& getProgram() { return program; }
 
 private:
-  void addInit(unsigned region, const Expr* addr, const llvm::Constant* val);
+  // 
+  // Pointer size: 32
+  //
+  static const unsigned ptrsize;
+  //
+  // A flag for bit-vector
+  //
+  static bool BIT_VECTOR;
```

Comment:
This variable name should not be capitalized since it is not a constant. Also, name this variable useBitVector, I think that would be more appropriate.